app/kumactl/cmd/install/install_metrics.go

```diff
@@ -26,12 +39,43 @@ func newInstallMetrics() *cobra.Command {
 		Short: "Install Metrics backend in Kubernetes cluster",
 		Long:  `Install Metrics backend (Prometheus and Grafana) in Kubernetes cluster.`,
 		RunE: func(cmd *cobra.Command, _ []string) error {
+			templateArgs := metricsTemplateArgs{
+				Namespace:               args.Namespace,
+				KumaPrometheusSdImage:   args.KumaPrometheusSdImage,
+				KumaPrometheusSdVersion: args.KumaPrometheusSdVersion,
+				KumaCpAddress:           args.KumaCpAddress,
+			}
+
 			templateFiles, err := data.ReadFiles(metrics.Templates)
 			if err != nil {
 				return errors.Wrap(err, "Failed to read template files")
 			}
+			var yamlTemplateFiles []data.File
+			for _, template := range templateFiles {
+				if strings.HasSuffix(template.Name, ".yaml") {
+					yamlTemplateFiles = append(yamlTemplateFiles, template)
+				}
+			}
```

Comment:
Could we please introduce a `type FileList []data.File` and define a `func (l FileList) Filter(predicate func(string) bool) FileList` operation on it ?